services/payments/src/simcore_service_payments/services/payments_gateway.py

```diff
@@ -0,0 +1,151 @@
+""" Interface to communicate with the payment's gateway
+
+- httpx client with base_url to PAYMENTS_GATEWAY_URL
+- Fake gateway service in services/payments/scripts/fake_payment_gateway.py
+
+"""
+
+
+import contextlib
+import logging
+from dataclasses import dataclass
+from typing import cast
+
+import httpx
+from fastapi import FastAPI
+from fastapi.encoders import jsonable_encoder
+from httpx import URL
+
+from ..core.settings import ApplicationSettings
+from ..models.payments_gateway import (
+    GetPaymentMethod,
+    InitPayment,
+    InitPaymentMethod,
+    PaymentID,
+    PaymentInitiated,
+    PaymentMethodID,
+    PaymentMethodInitiated,
+)
+
+_logger = logging.getLogger(__name__)
+
+
+@dataclass
+class PaymentGatewayApi:
```

Comment:
I see that instead of splitting into several modules as we usually do everything is attached to one class.
Any reason for this? You might still end up splitting it in the future.

Reply:
I wanted to have a single facade to communicate with the payments-gateway. 
My intention is to create several generic parts out of this  and put them in `servicelib.fastapi` in the near future